include/materials/tensor.h

```diff
@@ -0,0 +1,66 @@
+#ifndef MPM_TENSOR_H_
+#define MPM_TENSOR_H_
+
+#include <cmath>
+
+#include "Eigen/Dense"
+
+namespace mpm {
+namespace tensor {
+
+//! Compute mean stress p
+//! \param[in] stress Stress in Voigt notation where positive is tension
```

Comment:
Why is tension positive? In geotech applications most of our stresses are compressive

Reply:
I am open to discussion. My thought is to keep this tension positive for the purpose of other functions like computing deviatoric stress. In most geotech application, then users can multiply by -1 separately in material functions?

Reply:
The MPM code is mainly for Geotech applications, also I don't think we need to explicity define positive / negative

Reply:
OK let's get only `mean_p` positive, but the rest to be the same definition

Reply:
Done